test/command/suite/defrag/pat/clear_garbage/reuse_after_defrag.test

```diff
@@ -0,0 +1,34 @@
+# This is too slow with HTTP chunked.
+#@require-interface stdio
+
+table_create --name Users --flags TABLE_PAT_KEY --key_type ShortText
+
+#@disable-logging
+#@generate-series 1 260 Users '{"_key" => "A-User%04d" % i}'
+#@enable-logging
+delete Users --filter true
+
+check Users
+defrag
+check Users
+
+# Reuse targets were reset because defragmentation was done.
+# It is reused by adding and deleting again.
+#@disable-logging
+#@generate-series 1 260 Users '{"_key" => "A-User%04d" % i}'
+#@enable-logging
+delete Users --filter true
+
+# `_id = 261` is reused.
```

Comment:
How about explaining the node, which was `_id = 261, is reused because I thought `_id` was literarily resued?